tests/test_admin_functions.py

```diff
@@ -0,0 +1,83 @@
+from admin import *
+
+
+def test_add_drink():
+    drinks = {}
+    title = "Lager"
+    price = 34
+    add_drink(drinks, title, price)
+
+    assert title in drinks
+    assert drinks[title] == price
+
+
+def test_remove_drink():
+    drinks = {}
+    title = "Lager"
+    price = 40
+    add_drink(drinks, title, price)
```

Comment:
When you test the functionality of one function, it is not recommended to use other functions that you have created internally.
If the function used fails, the test will fail. But this will not mean that the test will fail because of a problem in the tested function.
Also, in unit tests, use only the function you are testing.